charts/stable/shiori/values.yaml

```diff
@@ -14,6 +14,14 @@ service:
         port: 10098
         targetPort: 8080
 
+installContainers:
+  initdb:
+    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+    volumeMounts:
+      - name: data
+        mountPath: "/data"
+    command: ["shiori", "migrate"]
```

Comment:
1. Does the migrate need to run only on install or on upgrades too?
2. the container will need access to the database too, so it needs it's env's etc

Reply:
only needs to run once.